test/typings/h-test.ts

```diff
@@ -0,0 +1,56 @@
+import { h, patch, Component } from "../../picodom";
+
+// empty vnode
```

Comment:
For this file I copied many of the assertions from the unit tests for `h` to validate that the typings cover all tested uses of `h`